tortoise/metrics.go

```diff
@@ -155,11 +155,46 @@ var (
 	executeEncodeVotes = encodeVotesHist.WithLabelValues("execute")
 )
 
-// LayerDistanceToBaseBallot checks how far back a node needs to find a good ballot.
-var layerDistanceToBaseBallot = metrics.NewHistogramWithBuckets(
-	"layer_distance_to_base_ballot",
-	namespace,
-	"How far back a node needs to find a good ballot",
-	[]string{},
-	prometheus.ExponentialBuckets(1, 2, 16),
-).WithLabelValues()
+var _ prometheus.Collector = (*collector)(nil)
```

Comment:
This isn't needed; if `collector` wouldn't implement `prometheus.Collector` line 161 would fail to compile with an error message that it doesn't 🙂